### ISSUE TITLE
Check 'signing' when building

### ISSUE THREAD

**Issue by johnbeisner** [dotnet/cli#7312]
A problem with signing is caught when inserting the CLI; we should catch it sooner - test the appropriate files every time we build.

**Comment by johnbeisner** [comment]
Note: Arcade should have a sign-check per @joeloff

**Comment by livarcocc** [comment]
Are we going to track this in Arcade or the CLI? If we feel this should be provided by arcade, the issue should probably be filled there.

**Comment by joeloff** [comment]
Yup, I know Core CLR is using it and ASP.NET as well. My SignCheck tool is integrated in Arcade AFAIK, it should just be a matter of having a repo leverage the validation step during the build
